generator/config/accumulator/first.yaml

```diff
@@ -13,3 +13,46 @@ arguments:
         name: expression
         type:
             - expression
+tests:
+    -
+        name: 'Use in $group Stage'
+        link: 'https://www.mongodb.com/docs/manual/reference/operator/aggregation/first/#use-in--group-stage'
+        pipeline:
+            -
+                $sort:
+                    item: 1
+                    date: 1
+            -
+                $group:
+                    _id: '$item'
+                    firstSale:
+                        $first: '$date'
+    -
+        name: 'Missing Data'
+        link: 'https://www.mongodb.com/docs/manual/reference/operator/aggregation/first/#missing-data'
+        pipeline:
+            -
+                $sort:
+                    item: 1
+                    price: 1
+            -
+                $group:
+                    _id: '$item'
+                    inStock:
+                        $first: '$quantity'
```

Comment:
This example is almost the same as the previous one. It makes sense in the documentation, but should we port all the examples into tests?

Reply:
I think the example makes sense in the docs to show behaviour of a missing field, but it's probably not relevant here. I'm fine removing it from the config, unless we want to make a point of having a builder version of each example in the documentation.

Reply:
OK to remove this. I don't see the point in including examples with no benefit to testing.